### ISSUE TITLE
matlab2tikz XTickLabel export

### ISSUE THREAD

**Issue by mschoeberl** [matlab2tikz/matlab2tikz#86]
Here is the problem - MWE:
```matlab
figure; plot(1:0.2:5, sin(1:0.2:5));
set(gca, 'XTick', 2:4)
set(gca, 'XTickLabel', {'first', 'second', 'third'} )
matlab2tikz('filename', 'testme.tikz', 'parseStrings', false, 'interpretTickLabelsAsTex', false);
```
Result: the export simply fails ... 

I can do some speculation: It looks like the array of cells is converted to a cell of cells in line 3598
```
for k = 1:length(tickLabel)
      tickLabel(k) = { prettyPrint( m2t, tickLabel(k), labelInterpreter) };
end
```
and the output is     {1x1 cell}   {1x1 cell}   {1x1 cell}
I tried a modification by changing this line to
```
for k = 1:length(tickLabel)
      tickLabel(k) = prettyPrint( m2t, tickLabel(k), labelInterpreter);
end
```
This seems to work for me, but I'm really not sure if this breaks anything else ...

**Comment by twippenbeck** [comment]
In the current dev version f5fc7dc this problem is existing anew.

Please use initial example above for reproduction.

Code was changed from loop + tickLabel{k} = ... to no loop and tickLabel = prettyPrint ...

Does not happen when switching parseString to true - but then latex commands can't be used successfully in many other places --> NOT a workaround.

Please solve !

Kind regards,

Tilman

**Comment by egeerardyn** [comment]
@twippenbeck Thank you for bringing this to our attention. We will check it and try to solve this.

**Comment by miscco** [comment]
I believe, there is a simple problem here:

```
    strings = cellstrOneLinePerCell(strings);
```
this overrides the original input with a cell containing individual cells rather than strings. The one sensible solution for parseStrings = false is to return earlier than that. So i would say we move the check that is in the loop for some reason and put it in front of the above line of code. That lets the code run.

**Comment by miscco** [comment]
Can you try #871?

**Comment by twippenbeck** [comment]
awesome !!! problem solved

great support & great project

thank you @miscco

**Comment by egeerardyn** [comment]
Fixed already by #871. Thank you, @miscco !
